src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneTransport.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\ContactEveryone;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Franck Ranaivo-Harisoa <franckranaivo@gmail.com>
+ */
+final class ContactEveryoneTransport extends AbstractTransport
+{
+    protected const HOST = 'contact-everyone.orange-business.com';
+
+    private string $token;
+    private ?string $diffusionName = null;
+    private ?string $category = null;
+
+
+    public function __construct(string $token, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->token = $token;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        $dsn = sprintf('contact-everyone://%s?token=%s', $this->getEndpoint(), $this->token);
```

Comment:
the token should not be included in the `__toString` output, to avoid putting credentials in it, as done in other bridges